### PR TITLE
Update the css-typed-om IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

File: interfaces/css-typed-om.idl
Comment (foolip) [inline interfaces/css-typed-om.idl]: This change blows up the test, there will be a single subtest failing with 'promise_test: Unhandled rejection with value: "CSS: namespace not yet supported"'. Blocked on https://github.com/w3c/web-platform-tests/issues/7583 in other words, although we can just back out this change and take the others.

**Review by foolip — CHANGES_REQUESTED** [review]

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25573)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350561493)

File: interfaces/css-typed-om.idl
Comment (lukebjerring) [reply interfaces/css-typed-om.idl]: Let's unblock via manual curation, then, and look at the blocker later.

**Comment by lukebjerring** [comment]
PTAL

**Review by foolip — APPROVED** [review]

**Pull request merged**
